Snowflake.Data/Core/FileTransfer/StorageClient/SFGCSClient.cs

```diff
@@ -422,7 +422,7 @@ public void DownloadFile(SFFileMetadata fileMetadata, string fullDstPath, int ma
                 }
                 else
                 {
-                    request = (HttpWebRequest)WebRequest.Create(generateFileURL(fileMetadata.stageInfo.location, fileMetadata.destFileName));
+                    request = (HttpWebRequest)WebRequest.Create(generateFileURL(fileMetadata.stageInfo.location, fileMetadata.srcFileName));
```

Comment:
cannot we use a method RemoveFileName() like above?

Reply:
I would leave it as is. It should be the same as RemoteFileName(), but this form would be more consistent with other cloud platform clients.